src/node/convolver.rs

```diff
@@ -359,6 +344,13 @@ impl ConvolverRendererInner {
         }
     }
 
+    // dummy convolver used to init renderer
+    fn tombstone() -> Self {
+        // just use arbitrary common sample rate
+        let padded_buffer = AudioBuffer::from(vec![vec![0.; 0]; 1], 48000.);
```

Comment:
This allocates (non-empty vec). I think you can reuse the tombstone from `AudioBufferSourceRenderer`

Reply:
Found a way to make it more clean in any case